Assets/_BForBoss/_Utility/Scripts/FreeRoamCamera.cs

```diff
@@ -103,10 +103,13 @@ public void Initialize(Transform playerTransform, Action onExit)
             _startingTransform = playerTransform;
             transform.SetPositionAndRotation(playerTransform.position, playerTransform.rotation);
             _onExitCamera = onExit;
-            _targetCameraState.SetFromTransform(transform);
-            _interpolatingCameraState.SetFromTransform(transform);
         }
-        
+
+        private void OnEnable()
```

Comment:
When you first open the FreeRoamCamera, it will spawn at 0,0,0 not at the Player's position. This fixes that